spec/system/tasks_system_spec.rb

```diff
@@ -87,7 +88,9 @@ def displayed_task_rating
         find_by_id('ratingModalOpenButton').click
 
         Rating::CATEGORIES.each do |category|
-          first(".task-star-rating[data-rating-category=#{category}][data-is-rating-input=true] .rating-star[data-rating='#{new_rating[category]}']", minimum: 0, wait: false)&.click
+          next unless new_rating.fetch(category, 0).positive?
+
+          first(".task-star-rating[data-rating-category=#{category}][data-is-rating-input=true] .rating-star[data-rating='#{new_rating[category]}']")&.click
```

Comment:
Safe navigation may not be needed anymore? (...and tbh feels a bit weird in specs)

Reply:
I was checking this code explicitly during the upgrade. It is something I overlooked when the feature was merged, but I considered a more detail change out of scope for the Rails upgrade (this PR).

The issue is that the subject (which this snippet is a part of) is very general and also tests invalid input. Specifically, it tries to click the "zeroth" out of 5 stars for some selection. As you can imagine, it is not possible to click that (but rather the first, second, ... fifth star). After some initial try and error, I opted to limit the scope of this PR and keep the current implementation.